spec/integration/s3_spec.rb

```diff
@@ -0,0 +1,100 @@
+require "logstash/devutils/rspec/spec_helper"
+require "logstash/outputs/s3"
+require 'socket'
+require "aws-sdk"
+require "fileutils"
+require "stud/temporary"
+
+describe LogStash::Outputs::S3, :integration => true, :s3 => true do
+  let!(:minimal_settings)  {  { "access_key_id" => ENV['AWS_ACCESS_KEY_ID'],
```

Comment:
Can we add, for example in a simple readme, that for testing we need to create env variables like described here? It's important for example when we try to test all plugins together.